lib/validate_partitions.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+package validate_partitions;
+
+use base "opensusebasetest";
+
+use strict;
+use warnings;
+use scheduler 'get_test_suite_data';
+use testapi;
+
+my $test_data = get_test_suite_data();
```

Comment:
you just need `table_type` as param to `validate_partition_table`, not the whole test data.

Reply:
That it, if we chose to add table_type in the test data, which is not the case currently.

Reply:
As we would have to create a lot of different yaml files for btrfs tests. So it's either compexity or lots of yamls.